EdFi.Ods.Utilities.Migration.Tests/MsSql/MigrationTests/v33_to_v34/V33ToV34SqlServerMigrationTest.cs

```diff
@@ -47,5 +53,23 @@ protected OdsUpgradeResult PerformTestMigration(string sourceDataScriptName = nu
         {
             return GetTableContents<T>(ToVersion);
         }
+
+        [Test]
+        public void ValidateJournalEntries()
+        {
+            var databaseReferencesJournalEntries = FetchDatabaseReferencesJournalEntries().ToList();
+
+            PerformTestMigration();
+
+            var deployJournalFullList = GetTableContents<DeployJournal>("[dbo].[DeployJournal]").Select(
+                x => x.ScriptName);
+
+            var deployJournal34List = deployJournalFullList.Where(y => databaseReferencesJournalEntries
+                    .Any(z => z.Contains(y,StringComparison.OrdinalIgnoreCase)))
+                    .ToList();
+
+            databaseReferencesJournalEntries.ToHashSet().SetEquals(deployJournal34List.ToHashSet()).ShouldBeTrue(
```

Comment:
I don't think this is right. If I change one of the deploy journal migration script entries or add one the test is still passing.

You may be able to skip filtering everything into the version specific list `deployJournal34List` and using your hashsets just check if `databaseReferencesJournalEntries` is a subset of `deployJournalFullList`.

Reply:
yes I updated . Thanks